### ISSUE TITLE
Paraphrase giving incorrect results after following the example

### ISSUE THREAD

**Issue by Rajmehta123** [deepmipt/DeepPavlov#1346]
### Apology for the image magnification. Couldn't scale down the image/screenshot/output.

**DeepPavlov version** (you can look it up by running `pip show deeppavlov`):
_0.12.1_

**Python version**:
_Python 3.7.0_

**Operating system** (ubuntu linux, windows, ...):
Linux

**Issue**:
Downloaded the _Quora question pairs dataset_ example by downloading the pretrained model file. https://deeppavlov.readthedocs.io/en/master/features/models/neural_ranking.html
Running the following example yields the wrong result. It should have been the paraphrase.
```
from deeppavlov import build_model, configs

para_model = build_model(configs.ranking.paraphrase_ident_qqp_interact, download=True)
para_model(['How can I be a good geologist?&What should I do to be a great geologist?'])
>>> 'This is a paraphrase.'
```
This is the output on my local.
![image](https://user-images.githubusercontent.com/22636443/99483589-887b1680-292c-11eb-81a8-191a7dbd112c.png)

**Error (including full traceback)**:

Incorrect result. No error.
![image](https://user-images.githubusercontent.com/22636443/99483599-8dd86100-292c-11eb-8561-003a2bc7d0d5.png)

**Comment by yurakuratov** [comment]
Hi!

On [this](https://deeppavlov.readthedocs.io/en/master/features/overview.html#ranking-model-docs) page you can find evaluation results for the model trained on QQP dataset. The model is not perfect (like any ML model) and has about ~87% accuracy on test set.

https://deeppavlov.readthedocs.io/en/master/features/overview.html#ranking-model-docs

**Comment by Rajmehta123** [comment]
But will it give different results on the same pair of sentences? Like the example on this page, https://deeppavlov.readthedocs.io/en/master/features/models/neural_ranking.html for the same sentence pair, it says 'This is paraphrase'

**Comment by puleon** [comment]
Hi!
Thank you for pointing us to this problem. We will fix it in the future release.

**Comment by puleon** [comment]
The model weights were corrupted. Please remove the folder `{MODELS_PATH}` with the old model. Typically, it is  `~/.deeppavlov/models/` folder. After that, download the model with new weights with the code line `para_model = build_model(configs.ranking.paraphrase_ident_qqp_interact, download=True)`. Everything should be fine. I close this issue.
